src/cljs/athens/devcards/breadcrumbs.cljs

```diff
@@ -49,42 +54,88 @@
 
 
 (defn breadcrumbs-list
-  [& children]
-  (into [:ol (use-style breadcrumbs-list-style) children]))
+  [{:keys [style]} & children]
+  (into [:ol (use-style (merge breadcrumbs-list-style style)) children]))
 
 
 (defn breadcrumb
-  [{:keys [key]} & label]
-  [:li (use-style breadcrumb-style {:key key})
-   [:a {:href "#"} label]])
+  [{:keys [style on-click]} & label]
+  [:li (use-style (merge breadcrumb-style style) {:title label})
+   [:a {:on-click on-click} label]])
 
 
 ;;; Devcards
 
 
 (defcard-rg Normal-Breadcrumb
-  [breadcrumbs-list
+  [breadcrumbs-list {}
```

Comment:
I think one way to let developer not have to pass in an empty property map is to give one- and two-arity signatures to the components.

```
(defn breadcrumbs-list
  ([& children]
   (into [:ol (use-style breadcrumbs-list-style) children]))
  ([{:keys [style]} & children]
   (into [:ol (use-style (merge breadcrumbs-list-style style)) children])))
```

A pattern I often see with these kinds of simple lists, is to pass in the list data to the main parent component. It might look like:

```
(def data ["Lorem" "Ipsum" "Athens"])
(defn component
  []
  [breadcrumbs-list (use-style {:color "blue"} {:data data})])
```

`breadcrumbs-list` is then in charge of iterating over the loop, and the API is simplified in a way. I'm not saying we should necessarily do this, just thinking about API design

Reply:
The way we are destructuring and then using keys is not ideal, especially in `button`. A better way might be something like

```
(defn button-primary
  [{:keys [style active class] :as props} & children]
  [:button (use-style
             (merge buttons-primary-style style)
             (merge (dissoc props [:style :active :class]) {:class (concat class (when active "active"))}))
   children])
```
Don't need to implement now, but I am requesting you do one-arity functions for these fixes.